tasks/connect_nfs.yml

```diff
@@ -0,0 +1,44 @@
+---
+# Configure NFS mounts
+
+- name: install nfs-common
+  become: yes
+  apt:
+    name: nfs-common
+    state: present
+
+- name: make nfs directories
+  become: yes
+  with_items:
+    - /var/www/files
+    - /var/www/files2
+  file:
+    state: directory
+    path: "{{ item }}"
+    owner: www-data
+    group: www-data
+    mode: 0664
+
+- name: configure nfs mount for files
+  become: yes
+  when: "{{ inventory_hostname not in groups.nfs }}"
+  mount:
+    src: "{{ nfs_server_for_files1 }}:/var/www/files"
```

Comment:
These need to be defined in the environment's `group_vars` directory. (I'm adding them)